fast/stages/01-resman/organization.tf

```diff
@@ -61,12 +66,16 @@ module "organization" {
       "roles/orgpolicy.policyAdmin" = local.branch_teams_pf_sa_iam_emails
     },
     local.billing_org ? {
-      "roles/billing.costsManager" = local.branch_teams_pf_sa_iam_emails
+      "roles/billing.costsManager" = concat(
+        local.branch_dataplatform_sa_iam_emails,
```

Comment:
I think you don't need this. Are you creating creating billing alerts?

Reply:
+1 let's remove and reintroduce later if needed